docs/src/main/asciidoc/security-openid-connect.adoc

```diff
@@ -179,7 +179,7 @@ NOTE: Adding a `%prod.` profile prefix to `quarkus.oidc.auth-server-url` ensures
 
 === Starting and Configuring the Keycloak Server
 
-NOTE: Do not start the Keycloak server when you run the application in a dev mode - `Dev Services for Keycloak` will launch a container. See <<keycloak-dev-mode, Running the Application in Dev mode>> section below for more information.
+NOTE: Do not start the Keycloak server when you run the application in a dev mode - `Dev Services for Keycloak` will launch a container. See <<keycloak-dev-mode, Running the Application in Dev mode>> section below for more information. For this example, make sure to put the {quickstarts-tree-url}/security-openid-connect-quickstart/config/quarkus-realm.json[realm configuration file] on the classpath so that it gets imported automatically when running dev mode.
```

Comment:
@aona42 If one starts from a ready quickstarts project then this realm is automatically added to the classpath. So I'd clarify it.

Reply:
@sberyozkin Hmm not sure, I usually start these guides from zero with a brand new project. What do you suggest?

Reply:
Yes, but the guide links to a complete solution, so if someone else go directly to this solution then we should advise it is not necessary to worry about dealing with this realm file, copying it to the classpath.

Reply:
@aona42 So I'd suggest is to add to your note one clarification, for example:
```
Make sure to put the {quickstarts-tree-url}/security-openid-connect-quickstart/config/quarkus-realm.json[realm configuration file] on the classpath (`target/classes` directory) so that it gets imported automatically when running in dev mode - unless you have already built a {quickstarts-tree-url}/security-openid-connect-quickstart[complete solution] in which case this realm file will be added to the classpath during the build.
```

Something along these lines...